cmake/modules/FindPortAudio.cmake

```diff
@@ -43,6 +43,8 @@ The following cache variables may also be set:
 
 #]=======================================================================]
 
+include(IsStaticLibrary)
```

Comment:
Note: I've added this include to be explicit, we previously assumed that it was transitively included through one of the other `FindX` modules.